### DISCUSSION TITLE
Welcome to exui Discussions!

### DISCUSSION THREAD

**Discussion by turboderp** [turboderp-org/exui#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bartowski1182** [comment]
Hey I'm bartowski, you may recognize me from TheBloke's discord, I like making docker files and lately started quanting some exl2s, got my automation nice and automated so it's a breeze now! All dockerized too :D

web ui is super slick, I like it a lot, very responsive! Would love to see someone put some effort into a mobile-friendly UI for it, that would be absolutely killer

**Reply by JohanAR** [reply]
I also made docker files. I'll attach them here if want to check if there's something worth merging with yours, in case you're going to make a PR. Might be possible to trim some stuff, copied a few lines from a different project, but I don't know when I'll have time to work more with it. Had to zip them for github to allow upload.

[exui_docker.zip](https://github.com/turboderp/exui/files/13209219/exui_docker.zip)

**Reply by bartowski1182** [reply]
Here's what i've got:

https://github.com/noneabove1182/exui-docker/tree/main

obviously anyone without repo access can't build right now, but when this repo is made public i'll update it so that it can pull directly from github the latest changes

**Reply by JohanAR** [reply]
Looks good! I didn't bother with a build step because I'm lazy :)

**Reply by bartowski1182** [reply]
Yeah I just do it for a smaller final build, devel is so massive
